### PR TITLE
Minor fixes to several b script tools

### PR DESCRIPTION
**Pull request by TrentHouliston**

### CODE REVIEW

File: nuclear/module/python/nuclear.py [hunk 1]
```diff
@@ -284,8 +284,8 @@ def Reactor(reactor):
             for include in reaction[1].include_paths():
                 includes.add('#include "{}"'.format(include))
 
-        macro_guard = "{}_HPP".format(reactor_name.upper())
-        header_file = "{}.hpp".format(reactor_name)
+        macro_guard = "{}_H".format(reactor_name.upper())
```
Comment (ElderNoSpace) [inline nuclear/module/python/nuclear.py:287]: Didn't we change to `.hpp` files everywhere?

**Review by ElderNoSpace — CHANGES_REQUESTED** [review]

File: nuclear/module/python/nuclear.py (hunk 1 above)
Comment (TrentHouliston) [reply nuclear/module/python/nuclear.py:287]: Yeah, you missed this one. Hence why it's a minor fixes PR :P.
Although the NUbots version of this file overrides it so this won't ever run anyway

Comment (ElderNoSpace) [reply nuclear/module/python/nuclear.py:287]: This diff is changing the `hpp` to `h` though. I think I missed it on the other repo.

Comment (JosephusPaye) [reply nuclear/module/python/nuclear.py:287]: Would be better to just fix it while we are here.

File: nuclear/module/python/nuclear.py [hunk 2]
```diff
@@ -326,7 +326,7 @@ class {class_name} : public NUClear::Reactor {{
             """
         )
 
-        with open(os.getcwd() + os.sep + reactor_name + ".hpp", "w") as f:
+        with open(os.getcwd() + os.sep + reactor_name + ".h", "w") as f:
```
Comment (JosephusPaye) [inline nuclear/module/python/nuclear.py:329]:
```suggestion
        with open(os.getcwd() + os.sep + reactor_name + ".hpp", "w") as f:
```

File: nuclear/module/python/nuclear.py (hunk 1 above)
Comment (JosephusPaye) [inline nuclear/module/python/nuclear.py:287]:
```suggestion
        macro_guard = "{}_HPP".format(reactor_name.upper())
```

File: nuclear/module/python/nuclear.py [hunk 3]
```diff
@@ -284,8 +284,8 @@ def Reactor(reactor):
             for include in reaction[1].include_paths():
                 includes.add('#include "{}"'.format(include))
 
-        macro_guard = "{}_HPP".format(reactor_name.upper())
-        header_file = "{}.hpp".format(reactor_name)
+        macro_guard = "{}_H".format(reactor_name.upper())
+        header_file = "{}.h".format(reactor_name)
```
Comment (JosephusPaye) [inline nuclear/module/python/nuclear.py:288]:
```suggestion
        header_file = "{}.hpp".format(reactor_name)
```

File: nuclear/module/python/nuclear.py (hunk 1 above)
Comment (ElderNoSpace) [reply nuclear/module/python/nuclear.py:287]: Its correct here.

Comment (TrentHouliston) [reply nuclear/module/python/nuclear.py:287]: Oh, looks like I messed this one up

**Review by ElderNoSpace — DISMISSED** [review]
LGTM

**Review by ElderNoSpace — APPROVED** [review]
Still good

**Pull request merged**